leafmachine-demo/requirements.txt

```diff
@@ -0,0 +1,2 @@
+kafka-python==2.0.2
```

Comment:
there are issues with kafka-python, it is now recommended to use [kafka-python-ng](https://pypi.org/project/kafka-python-ng/)
See stackoverflow: https://stackoverflow.com/questions/77287622/modulenotfounderror-no-module-named-kafka-vendor-six-moves-in-dockerized-